src/ceph-volume/ceph_volume/util/encryption.py

```diff
@@ -23,7 +23,7 @@ def create_dmcrypt_key():
     )
     # The size of the key is defined in bits, so we must transform that
     # value to bytes (dividing by 8) because we read in bytes, not bits
-    random_string = os.urandom(dmcrypt_key_size / 8)
+    random_string = os.urandom(int(dmcrypt_key_size / 8))
```

Comment:
I think it's more Pythonic to use `//` (truncating division) instead of casting to an int later. That works in 2.7 too.

Reply:
Casting to an `int` here prevents someone defining a float in dmcrypt_key_size, in which case, the float would be preserved. I would agree that just using `//` would be preferable otherwise.